databases/week1/dbweek1.sql

```diff
@@ -0,0 +1,56 @@
+-- 1. Find out how many tasks are in the task table
+SELECT count(*) as "Tasks in total" 
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT count(*) as "tasks in total"
+FROM task
+WHERE due_date is NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE status_id IN (SELECT id FROM status WHERE name = "Done");
+
+-- 4. Find all the tasks that are not marked as done
+SELECT *
+FROM task
+WHERE status_id in (SELECT id FROM status WHERE name != "Done");
+
+-- 5. Get all the tasks, sorted with the most recently created first
+
+SELECT * 
+FROM task
+ORDER BY created DESC; 
+
+-- 6. Get the single most recently created task
+SELECT * 
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+
+-- 7. Get the title and due date of all tasks where the title or description contains database
+
+SELECT title, description, due_date
+FROM task
+WHERE title like "%database%" or description like "%database%";
+
+-- 8. Get the title and status (as text) of all tasks
+SELECT task.title, status.name
+FROM task 
+JOIN status ON task.status_id = status_id;
```

Comment:
A small fix led to getting a different result.
Did you test how many rows you are getting (105).
Modify line #42 as below instead
`JOIN status ON task.status_id = status.id;`